tests/test_squirrel.c

```diff
@@ -95,43 +100,43 @@ Test(squirrel,seasonal_const){
   };
 
   float sc = sqr_eval_seasonal_cons(y,dim);
-  cr_assert_float_eq(sc, 11.77921898938974646313234009332, 1e-6, "the value of seasonal costant must be 11.7792");
+  cr_expect_float_eq(sc, 11.77921898938974646313234009332, 1e-3,

```

Comment:
@abhimanyu232 See other comment.

Reply:
Note that I did not change `1e-6` to `1e-3` on purpose, but using `FLT_EPSILON` breaks this test.